core/client

```diff
@@ -1 +1 @@
-Subproject commit 0a163d733315dfee2f5a20ea6aed41ffc7bc3bda
+Subproject commit 6af22d5b4244e2eac9d132c6412cb134f9775b06
```

Comment:
You need to remove this change from the PR for the tests to pass ;) Ghost-Admin is always updated posthumously 😁